app/includes/footer.njk

```diff
@@ -1,39 +1,44 @@
 {% from 'components/footer/macro.njk' import footer %}
+{% from 'components/panel/macro.njk' import panel %}
 
-{{ footer({
-  "list": "nhsuk-footer__list--three-columns",
-  "links": [
-    {
-      "URL": "https://www.nhs.uk/nhs-sites/",
-      "label": "NHS sites"
-    },
-    {
-      "URL": "https://www.nhs.uk/about-us/",
-      "label": "About us"
-    },
-    {
-      "URL": "https://www.nhs.uk/contact-us/",
-      "label": "Contact us"
-    },
-    {
-      "URL": "https://www.nhs.uk/personalisation/login.aspx",
-      "label": "Profile editor login"
-    },
-    {
-      "URL": "https://www.nhs.uk/about-us/sitemap/",
-      "label": "Sitemap"
-    },
-    {
-      "URL": "https://www.nhs.uk/accessibility/",
-      "label": "Accessibility"
-    },
-    {
-      "URL": "https://www.nhs.uk/our-policies/",
-      "label": "Our policies"
-    },
-    {
-      "URL": "https://www.nhs.uk/our-policies/cookies-policy/",
-      "label": "Cookies"
-    }
-  ]
-})}}
+<div data-test-id="footer">
+  <div data-test-id="footer-component">
+    {{ footer({
+      "list": "nhsuk-footer__list--three-columns",
+      "links": [
+        {
+          "label": "Buyer's Guide",
+          "URL": "/guide"
+        },
+        {
+          "label": "NHS Digital Helpdesk",
+          "URL": "/guide#contact-us"
+        },
+        {
+          "label": "NHS Digital",
+          "URL": "https://digital.nhs.uk/"
+        },
+        {
+          "label": "About GP IT Futures",
+          "URL": "https://digital.nhs.uk/services/future-gp-it-systems-and-services"
+        },
+        {
+          "label": "Capabilities & Standards Model",
+          "URL": "https://gpitbjss.atlassian.net/wiki/spaces/GPITF/overview"
+        }
+      ],
+      "copyright": "" 
+    })}}
+  </div>
+
+  <div data-test-id="legal-panel" class="bc-t-bg--grey-4 nhsuk-u-padding-bottom-3">
```

Comment:
let's add a tech debt task to add this to the BCL. We've got this legal panel here and in PB and most likely probably we'll need this is MP and Order From eventually.